starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,89 @@
+*{
+  font-family: sans-serif;
+}
+
+html{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+}
+
+h2{
+  margin-left: 15%;
+}
+
+ul {
+ counter-reset: custom;
+ list-style-type: none;
+ line-height: 1.6;
+ margin: 0 auto;
+}
+
+ul li:before {
+ content: '- ';
+}
+
+ol {
+ counter-reset: custom;
+ list-style-type: none;
+ line-height: 1.6;
+ margin: 0 auto;
+}
+
+ol li:before {
+ content: counter(custom)') ';
+ counter-increment: custom;
+}
+
+
+.apple-pies-image h1{
+  color: white;
+  font-size: 3em;
+  font-weight: bold;
+  text-align: center;
+  padding: 20%;
+}
+
+.apple-pies-image{
+  background-position: center;
+  background-image: url("../images/apple-pie.jpg");
+  background-repeat: no-repeat;
+  background-size: 65%;
```

Comment:
for this image it would have probably been better to set `background-size: cover`